packages/assets-controllers/CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added Base Network for networks to track in `TokenDetectionController` ([#5902](https://github.com/MetaMask/core/pull/5902))
   - Network changes were added in `@metamask/controller-utils`
+- Add `getErc20Balances` function within `TokenBalancesController` to support fetching ERC-20 token balances for a given address and token list ([#5925](https://github.com/MetaMask/core/pull/5925))
```

Comment:
Nit: can we add this Changelog before the  "- Added Base Network for network..."